environs/jujutest/livetests.go

```diff
@@ -220,7 +220,7 @@ func (t *LiveTests) BootstrapOnce(c *gc.C) {
 	// we could connect to (actual live tests, rather than local-only)
 	cons := constraints.MustParse("mem=2G")
 	if t.CanOpenState {
-		_, err := sync.Upload(t.toolsStorage, "released", nil, series.DefaultSupportedLTS())
+		_, err := sync.Upload(t.toolsStorage, "released", nil, version.DefaultSupportedLTS())
```

Comment:
Is this right? Diverges from `jujuversion` below.

Reply:
I'm waiting for CI tests to catch mistakes :cry: